winui/Masked-TextBox/Culture.md

```diff
@@ -1,10 +1,76 @@
 ---
 layout: post
-title: Localization in WinUI MaskedTextBox control | Syncfusion
-description: Learn all about the localization in the MaskedTextBox (SfMaskedTextBox) control for the specific set of symbols such as '.', ',', '/', ':' and '$'.
+title: Culture in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the culture in the MaskedTextBox (SfMaskedTextBox) control for the specific set of symbols such as '.', ',', '/', ':' and '$'.
 platform: WinUI
 control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Localization in WinUI SfMaskedTextBox
\ No newline at end of file
+# Culture in WinUI MaskedTextBox
+
+Special symbols like currency symbols, date separators, decimal separators, and others are applied based on the specified culture.
```

Comment:
@Vignesh-SF3580 The MaskedTextBox allows you to set any culture support for currency symbols, date separators, time separators, group separators,  and decimal separators by using the below special symbols.

Reply:
Changed the content.